.circleci/config.yml

```diff
@@ -316,7 +335,7 @@ workflows:
           name: publish-18-build-stack
           image_file: pack-18-build.tar
           image_tag: heroku/pack:18-build
-          image_tag_alias: heroku/pack:18-build
+          image_tag_alias: $REGISTRY_HOST/s/heroku/cnb/heroku:18-build
```

Comment:
What's the `/s/` in these aliases? stack?

Reply:
Since we all have`/u/`, for our own accounts I'm thinking it's short for user? Maybe `/s/` is service? Totally guessing 🤷🏻